src/js/post-911-gib-status/containers/IntroPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+
+import { getServiceUp } from '../actions/post-911-gib-status';
+import { SERVICE_UP_STATES } from '../utils/constants';
+
+export class IntroPage extends React.Component {
+  constructor(props) {
+    super(props);
+    // Make the api request
+    this.props.getServiceUp();
+  }
+
+
+  getContent() {
+    let content;
+    switch (this.props.serviceUp) {
+      case SERVICE_UP_STATES.unrequested: {
```

Comment:
Debating taking this out altogether, but it's here for full transparency on the lifecycle of the `serviceUp` prop.